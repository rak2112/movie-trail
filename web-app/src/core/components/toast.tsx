import React, { Children } from 'react';
import styled from 'react-emotion';
import { ToastContainer } from "react-toastify";
import "react-toastify/dist/ReactToastify.css";

const Toast = styled('div')`
  .Toastify__toast--success {
    background: #cc0000;
    opacity: 0.95;
    text-align: center;
    font-size: 16px;
    border-radius: 6px;
  }
`;


export const Toaster = ({duration}: { duration: number }) => {
  return (
    <Toast>
       <ToastContainer autoClose={duration}/>
    </Toast>
  )
};
