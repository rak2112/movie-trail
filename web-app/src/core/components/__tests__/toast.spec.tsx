import React from 'react';
import { create } from 'react-test-renderer';
import { Toaster} from '../toast';

describe(`Toast Component`, () => {
  const defaultProps ={ duration: 200 };
  it(`should render`, () => {
    const wrapper = create(
      <Toaster {...defaultProps}/>
    ).toJSON();
    
    expect(wrapper).toMatchSnapshot();
  });

});