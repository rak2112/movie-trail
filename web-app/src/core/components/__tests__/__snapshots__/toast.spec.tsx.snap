// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Toast Component should render 1`] = `
<div
  className="css-182pldt"
>
  <div
    className="Toastify"
  />
</div>
`;
